website/docs/getting-started/setup/installation-guides/docker/README.md

```diff
@@ -5,172 +5,157 @@ sidebar_position: 1
-
-:::note
-You can expect one container running when using the `docker-compose ps` command. Something like:
+## Setup with docker-compose
 
-_#Container appsmith Running_
-:::
+The Appsmith Docker image contains all the components required to run within a single Docker container. All these multiple processes are managed by a [Supervisord](http://supervisord.org/) instance, which is a lightweight process manager.
 
 ### Docker-compose configuration
 
-:::info
-Currently, auto-update is disabled on the docker-compose file. If you want to enable auto-update for Appsmith, please uncomment all the commented lines in the docker-compose file.
-:::
+Click the link below to download the `docker-compose.yml` file, and place it into your Appsmith installation folder.
 
-Download the below `docker-compose.yml` file into the appsmith installation folder
-
-<a target="_blank" href='/img/docker-compose_(3).yml' download="docker-compose_(3).yml">docker-compose.yml (Click to Download)</a>
+<div class="downloadAssets">
+    <a target="_blank" href='/img/docker-compose_(3).yml' download="docker-compose_(3).yml">
+     <img src="/img/FileDownload.png" alt="Click to Download"/>
+    </a>
+</div>
 
-**or** run the following curl if you're on a remote machine:
+**OR,** run the following cURL:
 
 ```bash
 curl -L https://bit.ly/3AQzII6 -o $PWD/docker-compose.yml
 ```
 
-:::note
-For the **Business Edition** - change the **image** name from `appsmith-ce` to `appsmith-ee` for the `image:` key in the `docker-compose.yml` file.
-:::
+:::info
+**For Business Edition users:**
 
-This configuration runs an Appsmith instance and a Watchtower instance to keep Appsmith automatically up-to-date.
+1. Change the **image** name from `appsmith-ce` to `appsmith-ee` for the `image:` key in the `docker-compose.yml` file.
 
-:::note
-For the **Business Edition** - add your **license key** (`APPSMITH_LICENSE_KEY`) to the `environment:` key in the `docker-compose.yml` file.
+2. Add your **license key** (`APPSMITH_LICENSE_KEY`) to the `environment:` key in the `docker-compose.yml` file.
+
+To upgrade your existing Community Edition installation to the Business Edition, [follow these instructions](../../upgrade-to-business-edition/).
 :::
 
-Bring the docker container up by running the following command. (You may need to run as `sudo` if docker and docker-compose are not accessible by your user)
+Start up the Docker container by running the following command - You may need to run with `sudo` if docker and docker-compose aren't accessible by your user.
 
 ```bash
 docker-compose up -d
 ```
 
-If it is not available locally, the command above will download the Docker images and start the services. You can follow the logs with the following command:
+If the image doesn't exist locally, this command downloads the Docker images and starts its services. You can follow the logs with the following command:
```

Comment:
Attempted a bit of a compromise on this one, because our new grammar linter disallows using `(s)`, and the alternative would be quite wordy I think. Let me know if this detail is a sticking point.

> If the image doesn't exist locally, this command downloads the necessary Docker image and starts the container.